mentoring/pairing/rest.py

```diff
@@ -6,9 +6,9 @@
 
 class PairSerializer(serializers.HyperlinkedModelSerializer):
     mentor = serializers.PrimaryKeyRelatedField(
-        queryset=Participant.objects.all().filter(role=Participant.MENTOR))
+        queryset=Participant.objects.all().filter(is_mentor=True))
```

Comment:
I've used customer managers for this in the past, such as this example with ``AuthorManager`` and ``EditorManager``. It would change the code to:

```
queryset=Participant.objects.mentors.all()
```

but would add some boilerplate code to create ``MentorQuerySet``, ``MentorManager``, etc. If you find yourself writing a lot of mentor-only or participant-only code, you may want to look into customer managers, but it is overkill until then.

Reply:
I thought about allowing the REST API to have separate `/mentors` and `/learners` endpoints to load those two sets, but chose to portray the single list of participants that's stored in the DB instead and divide those as necessary in the UI code.  That makes sense with this change, where "mentor" and "learner" are really just aspects of the more important concept of "participant".  So the data structure is for a participant (with a few fields that only make sense for mentors or learners), rather than distinct data structures for the two.  To make a long story short, I don't expect a lot of mentor-only or learner-only code.  But TIL [managers](https://docs.djangoproject.com/en/3.1/topics/db/managers/) :)